src/sql/workbench/parts/notebook/cellViews/code.component.ts

```diff
@@ -229,6 +229,11 @@ export class CodeComponent extends AngularDisposable implements OnInit, OnChange
 			}
 		}));
 		this._register(this.model.layoutChanged(() => this._layoutEmitter.fire(), this));
+		this._register(this.cellModel.onExecutionStateChange(event => {
```

Comment:
What happens when a cell is on the bottom of the screen (but still on the screen) and we run just that cell? Does scrolling happen or no?